Hcal/include/Hcal/Event/HcalHit.h

```diff
@@ -216,7 +259,10 @@ class HcalHit : public ldmx::CalorimeterHit {
 
   double timeDiff_{-9999.};
   double position_{-9999.};
-  double isX_{-9999.};
+  int orientation_{-9999};
+  bool orientationX_{false};
+  bool orientationY_{false};
+  bool orientationZ_{false};
```

Comment:
Do we have to store these three? Is it not sufficient to store the enum and have the member functions return comparisons with that instead?